src/main/java/com/esri/samples/na/closest_facility_static/ClosestFacilityStaticSample.java

```diff
@@ -0,0 +1,329 @@
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.Scene;
+import javafx.scene.control.Alert;
+import javafx.scene.control.Button;
+import javafx.scene.control.ProgressIndicator;
+import javafx.scene.layout.Background;
+import javafx.scene.layout.BackgroundFill;
+import javafx.scene.layout.CornerRadii;
+import javafx.scene.layout.StackPane;
+import javafx.scene.layout.VBox;
+import javafx.scene.paint.Paint;
+import javafx.stage.Stage;
+
+import com.esri.arcgisruntime.concurrent.ListenableFuture;
+import com.esri.arcgisruntime.data.Feature;
+import com.esri.arcgisruntime.data.FeatureQueryResult;
+import com.esri.arcgisruntime.data.FeatureTable;
+import com.esri.arcgisruntime.data.QueryParameters;
+import com.esri.arcgisruntime.data.ServiceFeatureTable;
+import com.esri.arcgisruntime.geometry.Envelope;
+import com.esri.arcgisruntime.geometry.GeometryEngine;
+import com.esri.arcgisruntime.layers.FeatureLayer;
+import com.esri.arcgisruntime.loadable.LoadStatus;
+import com.esri.arcgisruntime.mapping.ArcGISMap;
+import com.esri.arcgisruntime.mapping.Basemap;
+import com.esri.arcgisruntime.mapping.view.Graphic;
+import com.esri.arcgisruntime.mapping.view.GraphicsOverlay;
+import com.esri.arcgisruntime.mapping.view.MapView;
+import com.esri.arcgisruntime.symbology.PictureMarkerSymbol;
+import com.esri.arcgisruntime.symbology.SimpleLineSymbol;
+import com.esri.arcgisruntime.symbology.SimpleRenderer;
+import com.esri.arcgisruntime.tasks.networkanalysis.ClosestFacilityParameters;
+import com.esri.arcgisruntime.tasks.networkanalysis.ClosestFacilityResult;
+import com.esri.arcgisruntime.tasks.networkanalysis.ClosestFacilityRoute;
+import com.esri.arcgisruntime.tasks.networkanalysis.ClosestFacilityTask;
+import com.esri.arcgisruntime.tasks.networkanalysis.Facility;
+import com.esri.arcgisruntime.tasks.networkanalysis.Incident;
+
+public class ClosestFacilityStaticSample extends Application {
+
+  private MapView mapView;
+
+  @Override
+  public void start(Stage stage) throws Exception {
+
+    try {
+      // create stack pane and application scene
+      StackPane stackPane = new StackPane();
+      Scene scene = new Scene(stackPane);
+
+      // set title, size, and add scene to stage
+      stage.setTitle("Closest Facility (Static) Sample");
+      stage.setWidth(800);
+      stage.setHeight(700);
+      stage.setScene(scene);
+      stage.show();
+
+      // create control panel
+      VBox controlsVBox = new VBox(6);
+      controlsVBox.setBackground(new Background(new BackgroundFill(Paint.valueOf("rgba(0,0,0,0.3)"), CornerRadii.EMPTY,
+              Insets.EMPTY)));
+      controlsVBox.setPadding(new Insets(10.0));
+      controlsVBox.setMaxSize(150, 50);
+
+      // create buttons
+      Button solveRoutesButton = new Button("Solve Routes");
+      solveRoutesButton.setMaxWidth(Double.MAX_VALUE);
+      solveRoutesButton.setDisable(true);
+      Button resetButton = new Button("Reset");
```

Comment:
This button seems unnecessary. It solves the route the same way each time.

Reply:
Looks like I have two options:
1. Make a single button that toggles between 'solve' and 'reset'
2. Just leave the 'solve' button and have the `GraphicsOverlay` clear at the beginning, so we have a few seconds of the routes disappearing and reappearing once the task is solved.

In terms of code complexity option 1 would require creating a 'state' for the sample, so added complexity. But I think it would make a nicer sample since it would feel more interactive. What are your thoughts?

Reply:
Sorry, I should have been more clear. Just have the sample solve once and then disable the single Solve button. The user doesn't need to be able to solve multiple times.